isotovideo

```diff
@@ -260,6 +260,9 @@ $bmwqemu::vars{TEST_GIT_HASH} = $test_git_hash;
 # is not supposed to talk to the backend directly
 ($cpid, $cfd) = commands::start_server($bmwqemu::vars{QEMUPORT} + 1);
 
+# set a default distribution if the tests don't have one
+$testapi::distri ||= distribution->new;
```

Comment:
There is no point in doing || - the only place that sets it normally is the main.pm